src/utils/staticTokenDefinition.ts

```diff
@@ -32,7 +32,7 @@ export class StaticTokenDefinition {
       'DGD',
       BigInt.fromI32(9)
     )
-    staticDefinitions.push(tokenDGD)
+    staticDefinitions[0] = tokenDGD
```

Comment:
Why this change?

Reply:
push was throwing an error, and since all of them are fixed and assigned in that file I went for that approach. Let me know if you find any reason and you want me to investigate further to keep using the push.

Reply:
No need, was just curious